README.md

```diff
@@ -599,12 +599,19 @@ Proc. Advanced Maui Optical and Space Surveillance (AMOS) Technologies Conferenc
 <br><br>
 <a name="datasets"></a>
 # Datasets and Simulators (sorted by topic)
-- [Several datasets from the Sensors group at INI](http://sensors.ini.uzh.ch/databases.html) (Institute of Neuroinformatics), Zurich:
-    - [DVS128 Dynamic Vision Sensor Silicon Retina data](https://sourceforge.net/p/jaer/wiki/AER%20data/)
+- [Datasets from the Sensors group at INI](http://sensors.ini.uzh.ch/databases.html) (Institute of Neuroinformatics), Zurich:
+    - DVS09 - 	DVS128 Dynamic Vision Sensor Silicon Retina
+    - DVSFLOW16 - 	DVS/DAVIS Optical Flow Dataset
+    - DVSACT16 -	DVS Datasets for Object Tracking, Action Recognition and Object Recognition
+    - PRED18 - 	VISUALISE Predator/Prey Dataset
+    - DDD17 - 	DAVIS Driving Dataset 2017
+    - ROSHAMBO17 - 	RoShamBo Rock Scissors Paper game DVS dataset
+    - DHPE17 - 	DAVIS Human Pose Estimation and Action Recognition
     
 ### Optical Flow
 - [DVS/DAVIS Optical Flow Dataset](https://docs.google.com/document/d/1r9sRYANGuDTUcfSSq-sL4sd79SfjHGNRul_10uztDaI/pub) associated to the paper [Rueckauer and Delbruck, FNINS 2016](#Rueckauer16fnins).
 - [Binas et. al. ICML 2017](#Binas17icml). *DDD17: End-To-End DAVIS Driving Dataset*.
+- [MVSEC The Multi Vehicle Stereo Event Camera Dataset](https://daniilidis-group.github.io/mvsec/) associated to the paper Zhu, A. Z., Thakur, D., Ozaslan, T., Pfrommer, B., Kumar, V., & Daniilidis, K. (2018). [The Multi Vehicle Stereo Event Camera Dataset: An Event Camera Dataset for 3D Perception](https://arxiv.org/abs/1801.10202). arXiv preprint arXiv:1801.10202.
```

Comment:
MVSEC does no have optical flow, I think it would be a better fit in the Visual Odometry and SLAM subcategory.

Reply:
Also, I would try to format the paper and title as in our dataset, for consistency.